modules/custom/az_paragraphs/az_paragraphs.libraries.yml

```diff
@@ -38,3 +38,8 @@ az_paragraphs.preview:
   dependencies:
     - az_barrio/global-styling
     - az_barrio/arizona-bootstrap
+az_paragraphs.az_html:
+  version: VERSION
+  header: true
```

Comment:
I know that the ticket says move JQuery up, but I don't think it is required to be in the header, I think just the dependency is enough?

Reply:
@trackleft I tested removing the header option, but it looks like JQuery still gets added near the closing tag for the body tag. It only seams to work for me when I add it to the header.

Reply:
That's so weird, we must have a bug in our JS somewhere that breaks jquery after it.

Reply:
Oops I read that comment incorrectly disregard my comment.about the bug in JS somewhere, that doesn't make any sense.